### ISSUE TITLE
Network Manager Issues on Rasbian 10

### ISSUE THREAD

**Issue by spavolko** [homeworkc/lazycast#72]
Hey there! Great work, project looks quite promising. 

One problem I've run into though; networking seems to have broken completely on my Pi 3B+ on Rasbian 10/Buster after trying to install Network Manager. It says 'device not ready' for both eth and wlan in Network Manager's dropdown, and while I can still connect to WiFi using the default "Wireless & Wired Networking", I now no longer get a DHCP address from my network and instead get stuck with an APIPA address.

Has this been tested on the latest release of Rasbian? If so, anything obvious that I might have missed?

Thanks again!

**Comment by homeworkc** [comment]
Yes. I've tested on 2020-02-13 and 2020-05-27 releases. I generally choose the "Image with desktop based on Debian Buster" version. Sounds like the "Wireless & Wired Networking" is not purged properly. You can probably fix DHCP by re-enabling systemd-resolved: ``sudo systemctl enable systemd-resolved``.
